src/components/option/style.scss

```diff
@@ -1,20 +1,48 @@
+
+
```

Comment:
```suggestion
```

Reply:
Please remove empty lines in this and the other css files

Reply:
Done, for some reason if I save a *.css file it adds empty lines... :/